modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/ObjectEntryResourceImpl.java

```diff
@@ -516,6 +556,21 @@ private long _getPrimaryKey(
 		return objectEntry.getObjectEntryId();
 	}
 
+	private ObjectDefinition _getRelatedObjectDefinition(
```

Comment:
Maybe we should move this method to a common class as it is being used in the `RelatedObjectEntryResourceImpl` class.

I don't remember if I moved already in my PR that is still pending to be merged :thinking: